connectors/sync_job_runner.py

```diff
@@ -137,8 +137,10 @@ async def execute(self):
             self.elastic_server = SyncOrchestrator(self.es_config, self.sync_job.logger)
 
             if job_type in [JobType.INCREMENTAL, JobType.FULL]:
+                self.sync_job.log_info(f"Executing {job_type.value} sync")
                 await self._execute_content_sync_job(job_type, bulk_options)
             elif job_type == JobType.ACCESS_CONTROL:
+                self.sync_job.log_info(f"Executing {job_type.value} sync")
```

Comment:
```suggestion
                self.sync_job.log_info("Executing access control sync")
```

Reply:
I'd like to keep it this way in case rename happens or something similar

Reply:
My point is `job_type.value` is `access_control`, not `access control`

Reply:
Ah, makes sense